sbt-app/src/sbt-test/plugins/plugin-dependency-override-setup/build.sbt

```diff
@@ -0,0 +1,34 @@
+lazy val plugin = (project in file("baseplugin"))
+  .settings(
+    version := "0.1.0",
+    name:= "base-plugin",
+    organization := "sbt.test"
```

Comment:
Could you use `com.example` for this purpose?

Reply:
Also I am not 100% sure if this also works in `project/plugins.sbt` for resolution side, but I think

```scala
ivyPaths := IvyPaths((baseDirectory in LocalRootProject).value, Some((target in LocalRootProject).value / "ivy-cache"))
```

will change the destination of `publishLocal` to a directory that will be cleaned up.

Reply:
Made the `com.example` change.
I tried using the ivyPaths one, but it seems that it doesn't affect the publishing location. I put it in as a project setting, but that changed it to the local project's target directory which I would then need to figure out how to access to import from the other project. I'm open to the idea, but I'm also happy with where it is now.